apps/site-landing/src/containers/CreateAccountBanner/styles.module.less

```diff
@@ -1,3 +1,29 @@
-#createAccountBanner.content{
+@import "~@thecointech/site-semantic-theme/variables.less";
+
+.content {
```

Comment:
Removed ID because we have multiple CreateAccount banners (even tho they aren't the same element, they could become so in future)